pom.xml

```diff
@@ -96,6 +96,8 @@
     <zstd-jni.version>1.5.2-1</zstd-jni.version>
     <kubernetes-client.version>6.7.0</kubernetes-client.version>
     <rocksdbjni.version>8.5.3</rocksdbjni.version>
+    <jackson.version>2.10.5</jackson.version>
```

Comment:
why choose such a low version

Reply:
> why choose such a low version
The version of `org. apache. hadoop. shaded. *. jackson` comes from Hadoop 3.2.4 now. In order to reduce its impact on the original framework. I directly import the jackson 2.5.1 from Hadoop 3.2.4.